app/src/main/java/info/anwesha/iitp/Auth/RegisterFragment.java

```diff
@@ -68,8 +68,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         TextView loginTextView = view.findViewById(R.id.login_textview);
         loginTextView.setOnClickListener(view12 -> loadFragment(new info.anwesha.iitp.Auth.LoginFragment()));
 
-        ImageView imageView = view.findViewById(R.id.register_image);
-        Glide.with(getContext()).load(R.drawable.celesta_logo_long_2).into(imageView);
+//        ImageView imageView = view.findViewById(R.id.register_image);
+//        Glide.with(getContext()).load(R.drawable.celesta_logo_long_2).into(imageView);
```

Comment:
@Kavya-24 Remove the unused import of Glide and ImageView as well and push again. This can resolve the CIbuilding error